.github/workflows/job-postcommit-java-vr-spark-java11.yml

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Java VR Spark Java11 PostCommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  java_vr_spark_java11:
+    name: Run Spark ValidatesRunner Java 11
+    runs-on: [self-hosted, ubuntu-20.04]
+    timeout-minutes: 270
+    steps:
+      - name: Get number of available processors
+        id: get_available_processors
+        shell: bash
+        run: |
+          CORES=$(grep -c ^processor /proc/cpuinfo)
+          BALANCE_CORES=$((3*CORES))
+          echo $BALANCE_CORES
+          echo "::set-output name=BALANCE_CORES::$BALANCE_CORES"
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Setup environment
+        uses: ./.github/actions/setup-self-hosted-action
+      - name: Spark 3 Test Jobs
+        shell: bash
+        run: |
+          ./gradlew :runners:spark:3:jar
+          sleep 5
```

Comment:
Why do we need a sleep statement here?

Reply:
It is just a precaution to wait for the next command to be run, do you think we should eliminate it? @damccorm

Reply:
Is there any comment? Do I remove the `sleep` ?

Reply:
Sorry for the slow response here - could you help me understand why we need this precaution? Will this command not block until the spark server is spun up?

Reply:
Yes, I tested again and at the beginning, the `sleep` it was a precaution to give time to run the first command and then the next one, but testing out the `sleep` is not necessary as you mentioned, I will remove it.

Reply:
Fix applied